### DISCUSSION TITLE
Solar Smart MicroInverter MaySun600

### DISCUSSION THREAD

**Discussion by JanusHL** [jasonacox/tinytuya#571] [Show and tell]
Hi community,
First of all give thanks to the developer of this great application.

Recently I've purchased This microinverter that uses SmartLife- Tuya APP and following the instruccions I've extracted de DPS values in order to get data from it.

If could be interesting to add this new device to the devices DPS list , here are those values:

**Maysun600 MicroInverter**
Version 3.3
```
dps: {
       "2":  Cumulative total energy
       "10": Current power (/100)
       "15": Maysun600
       "16": Serial number
       "102": DC current (/100)
       "103": DC Voltage (/10)
       "104": AC Voltage (/10)
       "105": AC Current (/100)
       "108": Percent control
       }
```

best regards
JanusHL

**Comment by pgolos21** [comment]
Hello I know this is a old post but does this grid tie inverter .have the option to control the output power? Dps 108. Is it just read or read and write? Thanks you.

**Comment by JanusHL** [comment]
I can't say how this parameters acts in the microinverter. Mine has a value of 80:
`{
                    "code": "percent_control",
                    "value": 80
                }`

and parameter itself is:

`"108": {
                "code": "percent_control",
                "type": "Integer",
                "values": {
                    "unit": "%",
                    "min": 1,
                    "max": 100,
                    "scale": 0,
                    "step": 1
                }`

**Reply by pgolos21** [reply]
Does your inverter only out put 480watts? And does the tuya app have a option to adjust it? Thank you so much.

**Reply by JanusHL** [reply]
I have the microinverter not conected now. But normally it did'nt arrive to the max output power.
The APP does not let you change this parameter, but you can try it using:

`
d=tinytuya.OutletDevice(MICROID, MICROIP, MICROKEY)
d.set_version(version)
d.set_socketTimeout = 2
d.set_socketRetryLimit = 2

d.set_value(108, newvalue)

`
After it you can read again dps values to see the change...
